tests/test_gbdesAstrometricFit.py

```diff
@@ -491,6 +492,16 @@ def test_make_outputs(self):
                 self.assertAlmostEqual(np.mean(dDec), 0)
                 self.assertAlmostEqual(np.std(dDec), 0)
 
+    def test_output_model_params(self):
+        """Test the optional model parameters and covariance output."""
+        modelParams = pd.DataFrame(self.outputs.modelParams)
+        # Check that DataFrame is the expected size.
+        shape = modelParams.shape
+        self.assertEqual(shape[0] + 4, shape[1])
+        # Check that covariance matrix is symmetric.
+        covariance = (modelParams.iloc[:, 4:]).to_numpy()
+        np.testing.assert_array_almost_equal(covariance, covariance.T)
```

Comment:
Why almost? Shouldn't they be identical, otherwise it's not a covariance matrix? Note that the default for this check is 1e-6 precision and that's way more than typical roundoff error.

Reply:
In the test, the max absolute difference is 2e-19, but the max relative difference is 2e-6, so it doesn't pass (for 2 out of 10614564 elements).

Reply:
You might want to use [assert_allclose](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.htm) instead then, since that allow for setting separate absolute and relative tolerances.